### ISSUE TITLE
ZeroDivisionError: float division by zero

### ISSUE THREAD

**Issue by netbja** [freqtrade/ftui#8]
Hi all,
Trying ftui give me ZeroDivisionError on my Debian 12.

```
Starting FTUI - preloading all dataframes....Traceback (most recent call last):
  File "/home/bot/Freqtrade/ftui/.venv/bin/ftui", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/bot/Freqtrade/ftui/.venv/lib/python3.11/site-packages/ftui/ftui.py", line 474, in main
    op_data = ftapp._get_open_trade_dataframe(cl)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/bot/Freqtrade/ftui/.venv/lib/python3.11/site-packages/ftui/ftui.py", line 160, in _get_open_trade_dataframe
    ((t["stop_loss_abs"] - t["open_rate"]) / t["stop_loss_abs"]) * 100, 2
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
```
ZeroDivisionError: float division by zero

Why in your README.md you run 
`$ pip3 install -e .` 
Because,  we don't have any  setup.py ou pyproject.toml after `pip3 install -r requirements.txt`.

**Comment by xmatthias** [comment]
most likely a duplicate of #7 .

For the `pip3 install -e .`  question - i'm not sure what you're referring to - ftui has both a pyproject.toml, as well as a requirements file.
If you don't see these - you've not cloned the repository correctly.

**Comment by froggleston** [comment]
Fixed in 0.1.5
